worker/lib/workers/testbot.ts

```diff
@@ -61,14 +75,14 @@ class TestBotWorker extends EventEmitter implements Leviathan.Worker {
 			this.dutLogStream = createWriteStream(dutSerialPath);
 			dutLog.pipe(this.dutLogStream);
 		}
-
-		await this.hatBoard.setVout(5);
-		await this.hatBoard.powerOn();
+		console.log('Trying to power on DUT...');
```

Comment:
Thanks for adding logging, Leviathan truly needs that. Any more logging opportunities we could find that we can add more context on. I think the next target to add more logging would be the `Waiting for device to be reachable step`.

Reply:
yeah good idea man - I might quickly add a bit more logging before we merge